tests/backup/backup_helper.go

```diff
@@ -2026,3 +2028,28 @@ func MapToKeyValueString(m map[string]string) string {
 	}
 	return strings.Join(pairs, ",")
 }
+
+// LicenseCountCheck verifies the license count for backup
+func LicenseCountCheck(ctx context.Context, licenseInspectRequestObject *api.LicenseInspectRequest, expectedLicenseCount int64) error {
```

Comment:
Better names (I feel):
function name: `LicenseConsumedCountVerify` (you can use `check` instead of `verify` if that feels right, as both are fine)
COmment: `LicenseConsumedCountVerify verifies the consumed license count for px-backup`
and the last argument name: `expectedLicenseConsumedCount`

Reply:
done